src/main.py

```diff
@@ -78,6 +79,42 @@ def load_with_usb():
     TO_RUN_IN_MAIN = start_kolibri_with_usb
 
 
+def run_js(js):
```

Comment:
Nice. I was going to add the same helper function. Nit: a better name could be `evaluate_javascript()`. 
```suggestion
def evaluate_javascript(js_code):
```